src/tests/issues.targets

```diff
@@ -2219,6 +2220,9 @@
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/Runtime_68568/Runtime_68568/*">
             <Issue>Tests coreclr's handling of switches on natively sized integers</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Directed/callconv/ThisCall/EmptyThisCall">
```

Comment:
It doesn't seem like a universal truth that a thiscall with 0 args should be disallowed. For example, MSVC seems to handle it fine as well.

Reply:
> It doesn't seem like a universal truth that a thiscall with 0 args should be disallowed. 

It is undefined behavior. MSVC does it because of historical reasons.

Reply:
It shouldn't be that hard of an issue and @vargaz or @lambdageek should be able to point you in the right direction. However, if you don't have the bandwidth, then please file and issue and assign it to me. I'll handle it.

Reply:
@vargaz @lambdageek any pointers here?

Reply:
I opened #72438 about it.